doc/compiling/windows.md

```diff
@@ -2,67 +2,82 @@
 
 ## Requirements
 
-- [Visual Studio 2015 or newer](https://visualstudio.microsoft.com)
-- [CMake](https://cmake.org/download/)
-- [vcpkg](https://github.com/Microsoft/vcpkg)
-- [Git](https://git-scm.com/downloads)
-
+-   [Visual Studio 2015 or newer](https://visualstudio.microsoft.com)
+-   [CMake](https://cmake.org/download/)
+-   [vcpkg](https://github.com/Microsoft/vcpkg)
+-   [Git](https://git-scm.com/downloads)
 
 ## Compiling and installing the dependencies
 
-It is highly recommended to use vcpkg as package manager.
+It is highly recommended to use vcpkg as package manager, installed in a location without spaces in the path for compatibility with CMake.
+
+```
+"C:\Program Filesvcpkg\vcpkg.exe" # bad, may have issues with CMake
+"C:\vcpkg\vcpkg.exe" # good, no spaces means fewer problems
+```
+
+If spaces are present, you may see errors like:
+
+```
+libtool:   error: 'Files/vcpkg/buildtrees/libiconv/x64-windows-dbg/lib/libcharset.la' is not a directory
+```
 
 After you successfully built vcpkg you can easily install the required libraries:
 ```powershell
```

Comment:
CMake will also do installation when you invoke it, so I believe this step is technically unnecessary.

Reply:
Could you clarify what "this step" is? Maybe it can be removed in a separate PR? I'm personally re-building Luanti on a new machine now, I can try doing it without "this step", once I know what it is ;)

Reply:
Running the vcpkg command to install dependencies. CMake will run it for you when you do a configuration.

Reply:
I think it's fine to leave it here. It doesn't cause any issues. It's just something to keep in mind if you weren't aware of it, since I think fewer steps here likely means less that can go wrong.